### PR TITLE
Update cluster-singleton.md

### PR DESCRIPTION
**Pull request by ghost**
Forgot a word

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @rv-mkirchner,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Review by ktoso — APPROVED** [review]
Thanks :)

**Comment by ktoso** [comment]
As small as the change is, we need the CLA to be signed -- it's pretty simple, you can confirm it online by clicking the above link

**Comment by raboof** [comment]
OK TO TEST

File: akka-docs/src/main/paradox/typed/cluster-singleton.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Java
 :  @@snip [ShardingCompileOnlyTest.java]($akka$/akka-cluster-sharding-typed/src/test/java/jdoc/akka/cluster/sharding/typed/ShardingCompileOnlyTest.java) { #counter }
 
 Then on every node in the cluster, or every node with a given role, use the `ClusterSingleton` extension
-to spawn the singleton. An instance will per data centre of the cluster:
+to spawn the singleton. An instance will spawn per data centre of the cluster:
```
Comment (patriknw) [inline akka-docs/src/main/paradox/typed/cluster-singleton.md]: "will be spawn"
or perhaps "will be started", to avoid repeating same word again

**Pull request closed without merging**